src/simplemath.cpp

```diff
@@ -0,0 +1,77 @@
+#include "platform.h"
+#ifdef NCNN_SIMPLEMATH
+#include <stdlib.h>
+#include<stdio.h>
+float absolute(float x)//����ֵ
```

Comment:
The encoding can be improved. UTF-8 encoding recommended. Spaces around `//` prefered.

i.e.
```cpp
float absolute(float x) // your_comment
```